### PR TITLE
Add blog article about OCI image annotations and Kubernetes

### PR DESCRIPTION
**Pull request by gursimarsm**
https://github.com/kubernetes/website/issues/38356

This is an article explaining Annotations in Kubernetes as suggested in issue#38356

### CODE REVIEW

**Comment by sftim** [comment]
/hold

This article is about https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/ whereas issue https://github.com/kubernetes/website/issues/38356 is calling for an article about https://github.com/opencontainers/image-spec/blob/main/annotations.md

We shouldn't get these mixed up! @gursimarsm - can you see the mistake?

**Comment by sftim** [comment]
@gursimarsm if you'd like to write about this but aren't sure on the tech side of container images, maybe we (Kubernetes project) can help introduce you to someone who does.

If that sounds like what you'd want, maybe you and they could even co-author an article?

**Comment by gursimarsm** [comment]
Sure, that sounds great @sftim

**Comment by divya-mohan0209** [comment]
Hello @gursimarsm & @sftim : Do we want to continue work over this PR? Or would @gursimarsm be opening another PR to address the feedback?

**Comment by sftim** [comment]
I'm happy to see further work either in this PR or a different one.

File: content/en/blog/_posts/Annotations-in-Kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+layout: blog
+title: "Annotations in Kubernetes"
+slug: Annotations-in-Kubernetes
```
[the remark this answers is not in the document]
Comment (reylejano) [reply content/en/blog/_posts/Annotations-in-Kubernetes.md:4]: @gursimarsm , this blog requires a future date to be published, do you have time to propose a publication date

**Comment by kbhawkey** [comment]
@gursimarsm are you actively working on this blog article? If not, I can close the pull request.
Thanks for contributing.

**Comment by gursimarsm** [comment]
@kbhawkey I’ve been caught up with work. I’ll submit it by EoW. I hope that works

**Pull request closed without merging**